2022/Q2/artifacts/pos-model/PoS-model-cubic.md

```diff
@@ -88,8 +95,11 @@ tx_become_validator(validator_address, consensus_key, staking_reward_address)
     //set status to candidate and consensus key at n + pipeline_length
     validators[validator_address].consensus_key[cur_epoch+pipeline_length] = consensus_key
     validators[validator_address].state[cur_epoch+pipeline_length] = candidate
-    validators[validator_address].jailed = false
+    validators[validator_address].jail_record = JailRecord{is_jailed: false, epoch: ⊥}
     validators[validator_address].frozen = false
+    //add validator to the inactive set
+    add_validator_to_inactive(validator_address, pipeline_length)
```

Comment:
maybe this could use this function instead?
```suggestion
    add_validator_to_sets(validator_address, pipeline_length)
```

Reply:
Yes, I think so :)